app/views/home_buying_checklist/show.html.erb

```diff
@@ -0,0 +1,371 @@
+          <div class="home-buying-checklist__form-label">
+            <span class="visually-hidden">Cost 8 of 9</span> Removal costs
+          </div>
+        </div>
+
+        <div class="home-buying-checklist__form-center">
+          <div class="home-buying-checklist__form-description">
+            <p>The cost of hiring a removals company to help you move your possessions, or of hiring a van.</p>
+            <p>Average removal costs: £300 - £600</p>
+            <div class="home-buying-checklist__form-description--link">
+              <a href="https://www.moneyadviceservice.org.uk/en/articles/estimate-your-overall-buying-and-moving-costs">Further information</a>
+            </div>
+          </div>
+        </div>
+
+        <div class="home-buying-checklist__form-right">
+          £ <input type="number" class="home-buying-checklist__form-input" data-dough-input-value data-dough-input-removal>
+          <div class="home-buying-checklist__callout-inactive" data-dough-input-message>
+            <div class="callout">
+              <p>The average amount spent on removal costs is £300 - £600.</p>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="home-buying-checklist__form-row">
+        <div class="home-buying-checklist__form-left">
+          <div class="home-buying-checklist__form-label">
+            <span class="visually-hidden">Cost 9 of 9</span> Mortgage fee
+          </div>
+        </div>
+
+        <div class="home-buying-checklist__form-center">
+          <div class="home-buying-checklist__form-description">
+            <p>A fee you pay to your lender for arranging the mortgage. It can also be called a product fee, booking fee or application fee. Fee sizes vary depending on your mortgage.</p>
+            <p>Average fee: £100 – £2,250</p>
+            <div class="home-buying-checklist__form-description--link">
+              <a href="https://www.moneyadviceservice.org.uk/en/articles/estimate-your-overall-buying-and-moving-costs">Read more</a>
+            </div>
+          </div>
+        </div>
+
+        <div class="home-buying-checklist__form-right">
+          £ <input type="number" class="home-buying-checklist__form-input" data-dough-input-value data-dough-input-mortgage>
+          <div class="home-buying-checklist__callout-inactive" data-dough-input-message>
+            <div class="callout">
+              <p>The average mortgage fee is £100 - £2,250 but varies depending on your mortgage.</p>
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <div class="home-buying-checklist__form-row--gray">
+        <div class="home-buying-checklist__form-results-left">
+          <div class="home-buying-checklist__form-label">
+            Your total upfront cost:
+          </div>
+        </div>
+
+        <div class="home-buying-checklist__form-results-right" aria-live="polite">
+          £ <span data-dough-input-result>0</span>
```

Comment:
We've tested this with a screen reader and made some changes.

It seems to be fine to use `aria-live` on a div - it doesn't need to be a span. You're totally right about ensuring it doesn't just read the number - we moved the `aria-live` to the parent element, so now it reads "Your total upfront cost: X pounds" whenever it changes. Setting `aria-atomic` to true ensures that it reads the entire contents of the div instead of just the element that updates.

Every day is a school day - it's been interesting to learn more about screen reader accessibility.